test/Math/Quaternion.cpp

```diff
@@ -0,0 +1,98 @@
+#include <gtest/gtest.h>
+#include <lug/Math/Quaternion.hpp>
+#include <lug/Math/Geometry/Trigonometry.hpp>
```

Comment:
`#include <lug/Math/Geometry/Trigonometry.hpp>` should be before `#include <lug/Math/Quaternion.hpp>`